tests/SpecTests/RetryableReadsSpecTest.php

```diff
@@ -14,6 +14,7 @@
  * Retryable reads spec tests.
  *
  * @see https://github.com/mongodb/specifications/tree/master/source/retryable-reads
+ * @group serverless
```

Comment:
Do these tests actually pass? I assumed there might still be problems given [DRIVERS-1869](https://jira.mongodb.org/browse/DRIVERS-1869).

Reply:
Yes, the pass in the latest [Evergreen Patch](https://spruce.mongodb.com/task/mongo_php_library_serverless__php_edge_versions~latest_stable_driver_versions~latest_stable_test_serverless_patch_60bd25a16ae3b78b08d03b74de05282d2591fd3f_6114cbd13627e074e76eb764_21_08_12_07_20_50/tests).
The last build fails on all non-serverless builds with the same test failure, but I think that is unrelated.